kyuubi-ha/src/main/scala/org/apache/kyuubi/ha/client/ServiceDiscovery.scala

```diff
@@ -63,9 +63,22 @@ abstract class ServiceDiscovery(
 
   // stop the server genteelly
   def stopGracefully(isLost: Boolean = false): Unit = {
-    while (fe.be != null && fe.be.sessionManager.getOpenSessionCount > 0) {
-      debug(s"${fe.be.sessionManager.getOpenSessionCount} connection(s) are active, delay shutdown")
```

Comment:
just change to info ? the print interval is 1 minute, I think it does matter

Reply:
+1

Reply:
ok